src/components/AddDiveForm/AddDive.js

```diff
@@ -137,7 +139,9 @@ export default function AddDive({ handleTogglePage, onCreateDive }) {
             maxLength="500"
           ></textarea>
         </LastDivStyled>
-        <Button name="log dive">log dive</Button>
+        <ButtonSubmit variant="submit" name="log dive">
```

Comment:
You can use `attrs` here as well to have the `variant` backed into your styled component.